package.json

```diff
@@ -25,8 +25,12 @@
   "devDependencies": {
     "style-dictionary": "^2.8.1"
   },
+  "bin": {
+    "niten": "./bin/index.js"
```

Comment:
It is not yet clear how the flow will work.

Reply:
On Tanya, for example:
1. `npm i -g @getninjas/niten-tokens`
2. `niten --path "src/static/scss/gaiden"`
3. Import variables using "@import '../../gaiden/niten-tokens"

@williamcosta

Reply:
> On Tanya, for example:
> 
> 1. `npm i -g @getninjas/niten-tokens`
> 2. `niten --path "src/static/scss/gaiden"`
> 3. Import variables using "@import '../../gaiden/niten-tokens"
> 
> @williamcosta

Should this be in README?

Reply:
Yes! I'm gonna open another PR with that. Thanks for remind me 😄

Reply:
So we don't need these changes?

Reply:
We need it so we can install niten globally @williamcosta